### PR TITLE
🗃️ Add max_length validation for database models and input data

### PR DESCRIPTION
**Pull request by estebanx64**

### CODE REVIEW

File: backend/app/models.py [hunk 1]
```diff
@@ -1,43 +1,40 @@
...
     is_active: bool = True
     is_superuser: bool = False
-    full_name: str | None = None
+    full_name: str | None = Field(default=None, max_length=255)
 
 
 # Properties to receive via API on creation
 class UserCreate(UserBase):
-    password: str
+    password: str = Field(min_length=8, max_length=40)
```
Comment (tiangolo) [inline backend/app/models.py:15]: Could you please update the min length of passwords to the same of the other text fields, with 255?

**Review by tiangolo — COMMENTED** [review]
Nice! Just a small request. :pray:

File: backend/app/models.py (hunk 1 above)
Comment (PhilippWu) [reply backend/app/models.py:15]: I think it should be the max lenght?

Comment (tiangolo) [reply backend/app/models.py:15]: Yep, right @PhilippWu !

Comment (tiangolo) [reply backend/app/models.py:15]: Chatting internally, @estebanx64 convinced me to leave this with 40. :sweat_smile:

**Review by tiangolo — APPROVED** [review]
Nice, thanks! :rocket:

File: backend/app/models.py (hunk 1 above)
Comment (glendigity) [reply backend/app/models.py:15]: Hi @tiangolo, would you be able to share the logic on the 40? Sounds counterintuitive to me, so something to learn.

**Pull request merged**